frontend/src/pages/Assistants/Assistants.styling.tsx

```diff
@@ -1,30 +1,59 @@
 import styled from '@emotion/styled';
 import { Container } from '../../components/';
 
-const cardHeight = 216;
+const pxToRem = (px: number): string => `${px / 16}rem`;

```

Comment:
Btw, I'm not 100% sure here but since our font size does not change at all isn't this just cosmetics because it will be the same value on all screen sizes?

Reply:
I don't know how often people do it, but the idea is if someone has set different default font size than 16 in their browser, text and other things will scale accordingly.

Reply:
Oh yea, that information was stored somewhere in the depth of my brain.
Thanks for reminding me 🥇